tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/io/zip/StudentFileAwareZipper.java

```diff
@@ -129,4 +124,32 @@ private void writeToZip(Path currentPath, ZipArchiveOutputStream zipStream, Path
         log.trace("Closing entry");
         zipStream.closeArchiveEntry();
     }
+
+    private static String relativePathToZipCompliantName(Path path, boolean isDirectory) {
+        log.trace("Generating zip-compliant filename from Path \"{}\", isDirectory: {}",
+                path, isDirectory);
+        // zip standard says "/" is to be used as the path separator.
+        final char separator = '/';
```

Comment:
Can you move this out of the method and make it `private static final char ZIP_SEPARATOR = '/'`?